### ISSUE TITLE
A bug in neo_pyclone_annotation.py

### ISSUE THREAD

**Issue by songofbin** [bm2-lab/iTunes#5]
Some positive pMHC are missing in my pTuneos results. I think that's because of a bug in the 119 line of the neo_pyclone_annotation.py script. People should change it:

from
`data_drop=data_fill_na.drop_duplicates(subset=["Gene","MT_pep","WT_pep"])`
to
`data_drop=data_fill_na.drop_duplicates(subset=["HLA_type","Gene","MT_pep","WT_pep"])`

**Comment by ChiZhou-SITI** [comment]
Yes, if you want to get pMHC pairs from the results, HLA type should also be taken into consideration when drop duplicates. 
We will fix it later.
Thank you for your suggestion!
